src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -506,102 +699,8 @@ void EIRPlantLoopHeatPump::doPhysics(EnergyPlusData &state, Real64 currentLoad)
         this->fractionalDefrostTime = 0.0;
     }
     availableCapacity *= HeatingCapacityMultiplier;
-    this->defrostEnergy = this->defrostEnergyRate * reportingInterval;
```

Comment:
energy related report variables update has moved to the new member report() function.